install.sh

```diff
@@ -144,6 +144,10 @@ Install()
     # If update, start Wazuh
     if [ "X${update_only}" = "Xyes" ]; then
         WazuhUpgrade $INSTYPE
+        # Update versions previous to Wazuh 4.8
+        OSSEC_CONFIGURATION_FILE="$USER_DIR/etc/ossec.conf"
+        . ./src/init/replace_vulnerability_detector.sh
+        updateVulnerabilityDetector $OSSEC_CONFIGURATION_FILE
```

Comment:
Question: are the previous and current version numbers available on these scripts? I mean, could we somehow ask if we are upgrading to 4.8 or higher?

Reply:
only the previous version is available